Added support for Eventuate Local

dartpopikyardo opened the pull request.

dartpopikyardo committed:
added support for Eventuate Local for multi-module

dartpopikyardo committed:
fixed spring datasource url

dartpopikyardo committed:
fixed docker-compose scripts

dartpopikyardo committed:
added eventuateLocalVersion

dartpopikyardo committed:
- added Eventuate Local suport for single-module part

dartpopikyardo committed:
fixed logback files

cer commented on single-module/docker-compose.yml:
```diff
@@ -1,16 +1,15 @@
   environment:
-      EVENTUATE_API_KEY_ID:
-      EVENTUATE_API_KEY_SECRET:
+      EVENTUATE_API_KEY_ID: ${EVENTUATE_API_KEY_ID}
```
@dartpopikyardo This change is unnecessary

dartpopikyardo committed:
- fixed _build-and-test-all scripts: $EVENTUATE_LOCAL variable check
- removed unnecessary environment variables settings from docker-compose script

cer commented:
Merged manually
